.github/workflows/continuous-integration.yml

```
@@ -358,7 +373,7 @@ jobs:
           echo "DEMO_TARGET_EPOCH=$DEMO_TARGET_EPOCH" >> "$GITHUB_ENV"
 
       - name: Run node
-        timeout-minutes: ${{ matrix.network.timeout }}
+        timeout-minutes: ${{ env.TIMEOUT_RUN }}
```

actionlint (1.7.10)

type of expression at "float number value" must be number but found type string

(expression)